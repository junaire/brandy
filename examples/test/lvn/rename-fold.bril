# CMD: bril2json < {filename} | python3 ../../lvn.py -f | python3 ../../tdce.py tdce | bril2txt
@main {
  v1: int = const 4;
  v2: int = const 0;
  mul1: int = mul v1 v2;
  add1: int = add v1 v2;
  v2: int = const 3;
  print mul1;
  print add1;
}
