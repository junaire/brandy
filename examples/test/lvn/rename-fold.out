@main {
  mul1: int = const 0;
  add1: int = const 4;
  print mul1;
  print add1;
}
